net/samba4/Config.in

```diff
@@ -0,0 +1,111 @@
+if PACKAGE_samba4
+
+config SAMBA4_SERVER
+	bool "Server (fileserver)"
+	select BUSYBOX_CONFIG_RENICE
+	select BUSYBOX_DEFAULT_RENICE
+	help
+		installs: smbd (daemon) smbpasswd pdbedit testparm smbstatus smbtree
+		This provides the basic fileserver service and is the minimum needed to serve file-shares.
+		good guide: https://fitzcarraldoblog.wordpress.com/2016/10/17/a-correct-method-of-configuring-samba-for-browsing-smb-shares-in-a-home-network/
+	default y
+	
+	config SAMBA4_SERVER_NETBIOS
+		bool "enable Netbios support"
+		depends on SAMBA4_SERVER
+		help
+			installs: nmbd (daemon) nmblookup
+			Builds the nmbd to announce samba resources via netbios.
+			NOTE: Windows 10 (1709+) will not display workgroup netbios resources anymore, unless the old SMB1 feature is reinstalled (not recommend).
+			Therefor this feature can be safely disabled, network shares can still be directly accessed via network paths '\\hostname\sharename' and mounted via 'map network drive'.
+		default y	
+	
+	config SAMBA4_SERVER_VFS
+		bool "enable common VFS modules"
+		depends on SAMBA4_SERVER
+		help
+			provides commonly used vfs modules
+			installs: 
+				modules: vfs_btrfs vfs_fruit vfs_shadow_copy vfs_shadow_copy2 vfs_recycle vfs_fake_perms vfs_readonly vfs_cap vfs_offline vfs_crossrename
```

Comment:
vfs_btrfs should depend on kmod-fs-btrfs?
From what I can tell you only need vfs_shadow_copy2 not vfs_shadow_copy.
https://github.com/samba-team/samba/blob/master/source3/modules/vfs_shadow_copy.c
https://github.com/samba-team/samba/blob/master/source3/modules/vfs_shadow_copy2.c

Reply:
Yes for vfs_btrfs, will add a config check and i actually did not check for vfs_shadow_copy, but i think that was the samba default output via "smbd -b", so i kept it close to the default.